src/app/libp2p_helper/src/libp2p_helper/main.go

```diff
@@ -1208,20 +1233,12 @@ func gatingConfigFromJson(gc *setGatingConfigMsg) (*codanet.CodaGatingState, err
 	}
 	bannedPeers := peer.NewSet()
 	for _, peerID := range gc.BannedPeerIDs {
-		id, err := peer.Decode(peerID)
-		if err != nil {
-			logger.Errorf("error while parsing peer id %s: %v", peerID, err.Error())
-			continue
-		}
+		id := peer.ID(peerID)
```

Comment:
Yes, we should double check how this is encoded. I can check in my layover today if I have time.

Reply:
not sure this is right? we give string-encoded Peer.Id.t values directly to the helper in `mina_net2`